modules/xquare-user-interfaces/src/components/addonsitem/addonsitem.tsx

```
@@ -0,0 +1,207 @@
+
+const Status = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  margin-top: 6px;
+  gap: 25px;
+`;
+
+const Set = styled.div`
+  display: flex;
```

Biome (2.1.2)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)